globus_automate_client/cli/flows.py

```diff
@@ -940,15 +966,15 @@ def flow_action_log(
         callback=url_validator_callback,
     ),
     reverse: bool = typer.Option(
-        # Defaulting to any boolean value will reverse output - so we use None
-        None,
+        False,
```

Comment:
Based on how the API interprets any truthy value as a request to reverse the order, maybe we should just update the type annotation here that it can be a `None` value. Or we could implement the correct behavior here and try to update the API to behave correctly. That could be/should be a good story!

Reply:
I think I addressed this issue at its root failure point, in `globus_automate_client.action_client.ActionClient.log()`.

It should now be possible to use `True` and `False` throughout the code without special knowledge that only `None` actually works.